setup.py

```diff
@@ -0,0 +1,57 @@
+import codecs
+import os.path
+from setuptools import setup
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+
+def readme():
+    fname = os.path.join(here, "README.md")
+    with codecs.open(fname, encoding="utf8") as fptr:
+        return fptr.read()
+
+
+def version():
+    fname = os.path.join(here, "VERSION")
+    with open(fname, "r") as fptr:
+        return fptr.read().strip()
+
+
+reqs_list = ["numpy", "scipy"]
+
+pkg_dict = {"ibcdfo": ["pounders/py/*", "pounders/py/tests/regression_tests/*"]}
```

Comment:
I used these 
* `PkgData/*` - add in all internal data files whose contents could be loaded through the package interface
* `TestData/*` - add in all internal data files whose contents are consumed by the testing built into the package.

Do you need to declare your folders as part of this package?